azure-pipelines.yml

```diff
@@ -0,0 +1,43 @@
+variables:
+  Build.Repository.Clean: true
+  _HelixType: build/product
+  _HelixSource: pr/dotnet/arcade-services/$(Build.SourceBranch)
+
+# CI and PR triggers
+trigger:
+- master
+- switch-to-test-pool
+pr:
+- master
+
+# Three phases for each of the three OSes we want to run on
+phases:
+- template: /eng/common/templates/phases/base.yml
+  parameters:
+    name: Windows_NT
+    enableTelemetry: true
+    queue:
+      name: dotnet-external-temp
+      parallel: 99
+      matrix:
+        debug_configuration:
+          _BuildConfig: Debug
+        release_configuration:
+          _BuildConfig: Release
+    steps:
+    - script: eng\common\cibuild.cmd
+        -configuration $(_BuildConfig)
+        -prepareMachine
+      name: Build
+      displayName: Build
+      condition: succeeded()
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Logs to VSTS
```

Comment:
You need the "publish to BAR" step here: https://github.com/dotnet/arcade/blob/master/.vsts-ci.yml#L67

Reply:
Hrm... Do I need to do a Linux build here in addition to windows? I assume not, since it's just a managed library, but I don't really know how this is all hooked up.

Reply:
Yeah, I'd keep it.  Darc has a bunch of tests which would vary between OS's (line endings)

Reply:
Hrm. I'll probably need to chat with you or @chcosta in person about which parts of arcade are doing what... There are lots of bits and pieces of the yaml that look publishing related, but I'm not sure how many of them need to be copied, how many need to be moved, and how many I leave behind.

Reply:
I think I set the stuff I need up. Chris is apparently rewriting a lot of the publish stuff next week, so I want to put in what I've got so the build can actually start running, but I won't delete it from arcade until I also get the publish stuff working.